<!DOCTYPE html>
<html lang="en">
<meta name="viewport" content="width=device-width, initial-scale=1">
<head>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
  <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700&subset=latin,latin-ext' rel='stylesheet' type='text/css' />
  <link href='https://fonts.googleapis.com/css?family=Lato:400,300,300italic,700,400italic,700italic&subset=latin,latin-ext' rel='stylesheet' type='text/css' />

<style type='text/css'>
  #wsite-title {font-family:"Montserrat" !important;
  font-size: 15px !important;
  font-weight: 500;
  line-height: 1;
  text-transform: uppercase;
  letter-spacing: 0.08em;
  color:#000000;
}


.nav a {
  display: block;
  padding: 10px 20px;
  font-family: "brandon-grotesque", sans-serif;
  font-size: 14px;
  font-weight: 500;
  line-height: 1;
  letter-spacing: 0.05em;
  text-transform: uppercase;
  color:#000000;
}

.navbar .navbar-nav {
  display: inline-block;
  float: none;
}

.navbar .navbar-collapse {
  text-align: center;
}


.unite-header {
  position: relative;
  top: 0;
  z-index: 12;
  width: 100%;
  max-width: 100%;
  padding: 20px 40px;
  color: #000000;

}


.paragraph{
  font-family: "Lato" !important;
  color: #666666 !important;
  font-weight: 300;
  line-height: 1.75;
  letter-spacing: 0.02em;
  margin: 0 auto 30px;

}

.navbar-nav {
  float: right;
  margin: 0;
}

span.tab-space {padding-left:3em;}

.active{
  background-color: #ccc;
}

</style>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-123799212-1"></script>
<script>
window.dataLayer = window.dataLayer || [];
function gtag(){dataLayer.push(arguments);}
gtag('js', new Date());

gtag('config', 'UA-123799212-1');
</script>
<title>Prajwal Devkota, personal website</title>
</head>
<body>

  <!-- Fixed navbar -->
  <div class="unite-header"
  <nav class="navbar navbar-default navbar-fixed-top">
    <div class="container">
      <div class="navbar-header">
        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
          <span class="sr-only">Toggle navigation</span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </button>
        <a class="navbar-brand" href="/">
          <span id="wsite-title">Prajwal Devkota</span>
        </a>
      </div>
      <div id="navbar" class="collapse navbar-collapse">
        <ul class="nav navbar-nav">
          <li><a href="index.html">HOME</a></li>
          <li><a href="resume.html">RESUME</a></li>
          <li class="active"><a href="publications.html">PUBLICATIONS</a></li>
          <!-- <li><a href="#contact">PROJECTS</a></li>
          <li><a href="#contact">PHOTOGRAPHY</a></li> -->
          <li><a href="contact.html">CONTACT</a></li>
        </ul>
      </div><!--/.nav-collapse -->
    </div>
  </nav>
</div>
<!-- Begin page content -->
<div class="container" style="padding-top:60px;">










<font color="#818181" size="3">
<div class="row"  style="text-align:left; padding-top:1em">

        <div class="col-sm-1">
            <a href="javascript:myFunction('pap1')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            <font style="font-weight:bold;">Devkota, P.</font>, Wuchty, S., "Controllability of Regulatory Networks" (2021) <font style="font-weight:bold;">In Preparation</font>.
        </div>

<div id="pap1" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            Understanding the mechanism of transcriptional regulation at a tissue level is crucial as its mis-regulation can cause a broad range of diseases. Focusing on topological control of transcription factors through promoters and enhancers, we determined transcription factors that control increasing number of tissue specific networks through enhancer are enriched with known cancer genes, disease genes, drug targets and druggable proteins. Furthermore, target genes regulated through controller TF through promoter were significantly overrepresented in druggable and drug targets gene list. Among the druggable genes, G protein-couple receptors, ion channels, and transporter genes were enriched in at least one third of the tissue specific regulatory networks, indicating the control is exerted at the entry points in signaling pathways. Also, targets regulated through promoter controller TF were enriched with tissue specific and cancer specific genes. Focusing on transition between disease and healthy states, such target genes were also enriched with drug targets specific to cancer in the tissue, indicating fundamental role that control analysis of regulatory networks plays for our understanding of regulation and disease.
            <br /><br />
        </div>
</div>

</div>

<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap2')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            Johnson, N.F., N., Velasquez, Jha, O.,  Leahy, R., Johnson Restrepo,  N., Sear, R., Manrique, P., Lupu, Y.,   <font style="font-weight:bold;">Devkota, P.</font>, Wuchty, S., Goldberg, B. "Covid-19 infodemic reveals new tipping point epidemiology and a revised R formula" (2021) <font style="font-weight:bold;">In Submission</font>.
        </div>
</div>


<div id="pap2" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            Many governments have managed to control their COVID-19 outbreak with a simple message: keep the effective R  number R < 1 to prevent widespread contagion and flatten the curve. This raises the question whether a similar policy could control dangerous online'infodemics' of information, misinformation and disinformation. Here we show, using multi-platform data from the COVID-19 infodemic, that its online spreading instead encompasses a different dynamical regime where communities and users within and across independent platforms, sporadically form temporary active links on similar timescales to the viral spreading. This allows material that might have died out, to evolve and even mutate. This has enabled niche networks that were already successfully spreading hate and anti-vaccination material, to rapidly become global super-spreaders of narratives featuring fake COVID-19 treatments, anti-Asian sentiment and conspiracy theories. We derive new tools that incorporate these coupled social-viral dynamics, including an online R, to help prevent infodemic spreading at all scales: from spreading across platforms (eg Facebook, 4Chan) to spreading within a given subpopulation, or community, or topic. By accounting for similar social and viral timescales, the same mathematical theory also offers a quantitative description of other unconventional infection profiles such as rumors spreading in financial markets and colds spreading in schools.
            <br /><br />
        </div>
</div>


<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap3')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            <font style="font-weight:bold;">Devkota, P.*</font>, Manrique, P.*, Zheng, M., Johnson, N.F., Wuchty, S., "Resurgence of Online Hate Group Activity Reveals New Viral Epidemiology" (2021) <font style="font-weight:bold;">In Submission</font>.
        </div>
</div>


<div id="pap3" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            As social media platforms increasingly are used to propagate extremist content, understanding online patterns of extremism is of paramount importance to avoid such content to spread into the real world. Investigating the dynamics of responses to particularly controversial posts on Facebook pages that propagate extremist content, we observed resurgence patterns of activity that feature initial large and consecutive smaller peaks of activity that point to an ongoing discussion of the given post. We described such patterns with a novel Epidemiological Common Space Contagion model (ECSC) where contagion only can appear on social media pages that serve as common spaces where users get infected when they post a comment. Counterintuitively, we found that resurgence patterns intensify when the chance to prompt users to post a comment decreases as well as users spend less time on a page. As a corollary, we investigated intervention strategies, suggesting that a combination of curbing the access of new users to a page and keeping users that are already on a page long after the first peak of response activity effectively eradicated resurgence patterns.
            <br /><br />
        </div>
</div>

<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap4')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            Chamling, X., Kallman, A., Fang, W., Berlinicke, C., Mertz, J., <font style="font-weight:bold;">Devkota, P.</font>, Pantoja, I. M., Smith, M., Ji, Z., Chang, C., Kaushik, A., Chen, L., Whartenby K., Calabresi P. A., Mao, H., Ji, H., Wang, T., Zack, D.J. "Single-cell transcriptomic reveals molecular diversity and developmental heterogeneity of human stem cell-derived oligodendrocyte lineage cells" <font style="font-style:italic;">Nature Communications.</font> (2021).  <a href="https://rdcu.be/cel8Y" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>


<div id="pap4" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            Injury and loss of oligodendrocytes can cause demyelinating diseases such as multiple sclerosis. To improve our understanding of human oligodendrocyte development, which could facilitate development of remyelination-based treatment strategies, here we describe time-course single-cell-transcriptomic analysis of developing human stem cell-derived oligodendrocyte-lineage-cells (hOLLCs). The study includes hOLLCs derived from both genome engineered embryonic stem cell (ESC) reporter cells containing an Identification-and-Purification tag driven by the endogenous PDGFRα promoter and from unmodified induced pluripotent (iPS) cells. Our analysis uncovers substantial transcriptional heterogeneity of PDGFRα-lineage hOLLCs. We discover sub-populations of human oligodendrocyte progenitor cells (hOPCs) including a potential cytokine-responsive hOPC subset, and identify candidate regulatory genes/networks that define the identity of these sub-populations. Pseudotime trajectory analysis defines developmental pathways of oligodendrocytes vs astrocytes from PDGFRα-expressing hOPCs and predicts differentially expressed genes between the two lineages. In addition, pathway enrichment analysis followed by pharmacological intervention of these pathways confirm that mTOR and cholesterol biosynthesis signaling pathways are involved in maturation of oligodendrocytes from hOPCs.
            <br /><br />
        </div>
</div>

<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap5')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            <font style="font-weight:bold;">Devkota, P.</font>, Danzi, M., Lemmon, V., Bixby, J., Wuchty, S., "Computational Identification of Kinases that Control Axon Growth in Mouse," <font style="font-style:italic;">SLAS Discovery.</font> 2020. <a href="https://journals.sagepub.com/doi/abs/10.1177/2472555220930697" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>


<div id="pap5" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            The determination of signaling pathways and transcriptional networks that control various biological processes is a major challenge from both basic science and translational medicine perspectives. Because such analysis can point to critical disease driver nodes to target for therapeutic purposes, we combined data from phenotypic screening experiments and gene expression studies of mouse neurons to determine information flow through a molecular interaction network using a network propagation approach. We hypothesized that differences in information flow between control and injured conditions prioritize relevant driver nodes that cause this state change. Identifying paths likely taken from potential source nodes to a set of transcription factors (TFs), called sinks, we found that kinases are enriched among source genes sending significantly different amounts of information to TFs in an axonal injury model. Additionally, TFs found to be differentially active during axon growth were enriched in the set of sink genes that received significantly altered amounts of information from source genes. Notably, such enrichment levels hold even when restricting the set of source genes to only those kinases observed to support or hamper neurite growth. That way, we found a set of 71 source genes that send significantly different levels of information to axon growth-relevant TFs. We analyzed their information flow changes in response to axonal injury and their influences on TFs predicted to facilitate or antagonize axon growth. Finally, we drew a network diagram of the interactions and changes in information flow between these source genes and their axon growth-relevant sink TFs.
            <br /><br />
        </div>
</div>

<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap6')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            <font style="font-weight:bold;">Devkota, P.</font>, Wuchty, S., "Controllability analysis of molecular pathways points to proteins that control the entire interaction network"  <font style="font-style:italic;">Scientific reports.</font> (2020). <a href="https://www.nature.com/articles/s41598-020-59717-6" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>


<div id="pap6" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            Inputs to molecular pathways that are the backbone of cellular activity drive the cell to certain outcomes and phenotypes. Here, we investigated proteins that topologically controlled different human pathways represented as independent molecular interaction networks, suggesting that a minority of proteins control a high number of pathways and vice versa. Transcending different topological levels, proteins that controlled a large number of pathways also controlled a network of interactions when all pathways were combined. Furthermore, control proteins that were robust when interactions were rewired or inverted also increasingly controlled an increasing number of pathways. As for functional characteristics, such control proteins were enriched with regulatory and signaling genes, disease genes and drug targets. Focusing on evolutionary characteristics, proteins that controlled different pathways had a penchant to be evolutionarily conserved as equal counterparts in other organisms, indicating the fundamental role that control analysis of pathways plays for our understanding of regulation, disease and evolution.
            <br /><br />
        </div>
</div>

<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap7')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            Boltz, T. A., <font style="font-weight:bold;">Devkota, P.</font>, Wuchty, S., "Collective influencers in protein interaction networks."  <font style="font-style:italic;">Scientific reports.</font> (2019). <a href="https://www.nature.com/articles/s41598-019-40410-2" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>

<div id="pap7" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
            Recent research increasingly shows the relevance of network based approaches for our understanding of biological systems. Analyzing human protein interaction networks, we determined collective influencers (CI), defined as network nodes that damage the integrity of the underlying networks to the utmost degree. We found that CI proteins were enriched with essential, regulatory, signaling and disease genes as well as drug targets, indicating their biological significance. Also by focusing on different organisms, we found that CI proteins had a penchant to be evolutionarily conserved as CI proteins, indicating the fundamental role that collective influencers in protein interaction networks plays for our understanding of regulation, diseases and evolution.
            <br /><br />
        </div>
</div>




<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap8')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            Johnson, N.F., Leahy, R., Johnson Restrepo, N., Velasquez, N., Zheng, M., Manrique, P.,  <font style="font-weight:bold;">Devkota, P.</font>, Wuchty, S., "Hidden resilience and adaptive dynamics of the global online hate ecology" <font style="font-style:italic;">Nature</font> 573 (2019). <a href="https://www.nature.com/articles/s41586-019-1494-7" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>

<div id="pap8" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
  Online hate and extremist narratives have been linked to abhorrent real-world events, including a current surge in hate crimes and an alarming increase in youth suicides that result from social media vitriol; inciting mass shootings such as the 2019 attack in Christchurch, stabbings and bombings; recruitment of extremists, including entrapment and sex-trafficking of girls as fighter brides; threats against public figures, including the 2019 verbal attack against an anti-Brexit politician, and hybrid (racist–anti-women–anti-immigrant) hate threats against a US member of the British royal family; and renewed anti-western hate in the 2019 post-ISIS landscape associated with support for Osama Bin Laden’s son and Al Qaeda. Social media platforms seem to be losing the battle against online hate and urgently need new insights. Here we show that the key to understanding the resilience of online hate lies in its global network-of-network dynamics. Interconnected hate clusters form global ‘hate highways’ that—assisted by collective online adaptations—cross social media platforms, sometimes using ‘back doors’ even after being banned, as well as jumping between countries, continents and languages. Our mathematical model predicts that policing within a single platform (such as Facebook) can make matters worse, and will eventually generate global ‘dark pools’ in which online hate will flourish. We observe the current hate network rapidly rewiring and self-repairing at the micro level when attacked, in a way that mimics the formation of covalent bonds in chemistry. This understanding enables us to propose a policy matrix that can help to defeat online hate, classified by the preferred (or legally allowed) granularity of the intervention and top-down versus bottom-up nature. We provide quantitative assessments for the effects of each intervention. This policy matrix also offers a tool for tackling a broader class of illicit online behaviours such as financial fraud.
            <br /><br />
        </div>
</div>


<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap9')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            <font style="font-weight:bold;">Devkota, P.</font>, Danzi M, Wuchty, S., "Beyond degree and betweeness centrality: Alternative topological measures to predict viral targets," <font style="font-style:italic;">PloS-one</font> (2018) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0197595" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>



<div id="pap9" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
      The availability of large-scale screens of host-virus interaction interfaces enabled the topological analysis of viral protein targets of the host. In particular, host proteins that bind viral proteins are generally hubs and proteins with high betweenness centrality. Recently, other topological measures were introduced that a virus may tap to infect a host cell. Utilizing experimentally determined sets of human protein targets from Herpes, Hepatitis, HIV and Influenza, we pooled molecular interactions between proteins from different pathway databases. Apart from a protein’s degree and betweenness centrality, we considered a protein’s pathway participation, ability to topologically control a network and protein PageRank index. In particular, we found that proteins with increasing values of such measures tend to accumulate viral targets and distinguish viral targets from non-targets. Furthermore, all such topological measures strongly correlate with the occurrence of a given protein in different pathways. Building a random forest classifier that is based on such topological measures, we found that protein PageRank index had the highest impact on the classification of viral (non-)targets while proteins' ability to topologically control an interaction network played the least important role.
            <br /><br />
        </div>
</div>



<div class="row"  style="text-align:left; padding-top:1em">
        <div class="col-sm-1">
            <a href="javascript:myFunction('pap10')">Abstract</a><span class="tab-space ">
        </div>
        <div class="col-md-11">
            Goodacre, N., <font style="font-weight:bold;">Devkota, P.</font>, Bae, E., Wuchty, S., Uetz, P., "Protein-protein interactions of human viruses," <font style="font-style:italic;">Seminars in Cell &#38; Developmental Biology.</font> Academic Press, 2018. <a href="https://www.sciencedirect.com/science/article/abs/pii/S1084952117304998" target="_blank">Link</a><span class="tab-space ">
        </div>
</div>

<div id="pap10" style="display:none;">
        <div class="col-sm-1">
          <br /><br /><br /><br /><br /><br />
            <!-- Link<span class="tab-space "> -->
        </div>
        <div class="col-md-11" style="color:#444;">
          <br /><br />
      Viruses infect their human hosts by a series of interactions between viral and host proteins, indicating that detailed knowledge of such virus-host interaction interfaces are critical for our understanding of viral infection mechanisms, disease etiology and the development of new drugs. In this review, we primarily survey human host-virus interaction data that are available from public databases following the standardized PSI-MS format. Notably, available host-virus protein interaction information is strongly biased toward a small number of virus families including herpesviridae, papillomaviridae, orthomyxoviridae and retroviridae. While we explore the reliability and relevance of these protein interactions we also survey the current knowledge about viruses functional and topological targets. Furthermore, we assess emerging frontiers of host-virus protein interaction research, focusing on protein interaction interfaces of hosts that are infected by different viruses and viruses that infect multiple hosts. Finally, we cover the current status of research that investigates the relationships of virus-targeted host proteins to other comorbidities as well as the influence of host-virus protein interactions on human metabolism.
            <br /><br />
        </div>
</div>









</div>
<script src="https://code.jquery.com/jquery-1.12.0.min.js"></script>
<script>
function myFunction(divId) {
  var x = document.getElementById(divId);
  if (x.style.display === "none") {
    x.style.display = "block";
  } else {
    x.style.display = "none";
  }
}
</script>


</body>
</html>